test/asserts/equal-keys-assert_test.js

```diff
@@ -45,9 +45,49 @@ describe('EqualKeysAssert', () => {
     }
   });
 
+  it('should throw an error if the object is empty', () => {
+    try {
+      new Assert().EqualKeys(['foo']).validate({});
+
+      should.fail();
+    } catch (e) {
+      e.show().violation.difference.should.eql(['foo']);
+    }
+  });
+
+  it('should allow setting no keys', () => {
```

Comment:
What are we setting here? WDYT of rewording to something along the lines of:

    'should allow `keys` to be undefined'